django/test/testcases.py

```diff
@@ -730,6 +731,22 @@ def assertWarnsMessage(self, expected_warning, expected_message, *args, **kwargs
             *args, **kwargs
         )
 
+    @contextmanager
+    def assertNoLogs(self, logger_name, levelname='INFO'):
```

Comment:
Can we use the same signature as unittest's [assertNoLogs()](https://docs.python.org/3.10/library/unittest.html?highlight=assertnologs#unittest.TestCase.assertNoLogs)?
```suggestion
    def assertNoLogs(self, logger, level=None):
```